platform-sdk/swirlds-merkledb/src/test/java/com/swirlds/merkledb/collections/AbstractLongListTest.java

```diff
@@ -247,6 +262,114 @@ void writeReadEmptyList(@TempDir final Path tempDir) throws IOException {
         }
     }
 
+    @Test
+    void writeReadSizeOfOne(@TempDir final Path tempDir) throws IOException {
+        try (final AbstractLongList<?> list = createLongList()) {
+            list.updateValidRange(1, 1);
```

Comment:
A similar test is needed, when the valid range is limited to one chunk, but not the very first chunk. Probably, the test can be changed to @ParametrizedTest with start and end indices as parameters